src/parser.rs

```diff
@@ -144,9 +144,9 @@ impl<'source> Parser<'source> {
             let field = self.slice().trim_start_matches('%');
             if !seen.insert(field) {
                 return Err(ParserError::with_detail(
-                    ParserErrorKind::InvalidValue,
+                    ParserErrorKind::DuplicateField,
                     label.span(),
-                    format!("duplicate field {field:?}"),
+                    format!("{field:?}"),
```

Comment:
Probably don't really need these to be `Debug` since they're guaranteed to be valid labels at this point.

Reply:
I personally think the errors read better with some quotes; could change it to be in ``

Reply:
Nah its fine